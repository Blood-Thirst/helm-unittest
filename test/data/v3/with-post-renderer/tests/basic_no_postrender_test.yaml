templates:
  - templates/basic.yaml
tests:
  - it: manifest should match snapshot
    asserts:
      - matchSnapshot: {}
      - notExists:
          path: metadata.annotations.appended
      - equal:
          path: metadata.annotations.foo
          value: bar
