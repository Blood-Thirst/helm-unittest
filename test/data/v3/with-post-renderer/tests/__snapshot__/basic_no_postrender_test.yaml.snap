manifest should match snapshot:
  1: |
    metadata:
      annotations:
        foo: bar
